Dockerfile-metadata

```diff
@@ -2,26 +2,29 @@
 
 FROM arxiv/base
 
-RUN pip install uwsgi
+WORKDIR /opt/arxiv
+
+# We're using MariaDB for testing/development, but production is MySQL (for
```

Comment:
Just a caution that while MariaDB 10.3.x and MySQL 5.7.x are very similar, there are some subtle [differences](https://mariadb.com/kb/en/library/function-differences-between-mariadb-103-and-mysql-57/) (notably JSON); in all likelihood this won't be a problem for us, but something to watch out for.

Reply:
Good to know; at this point this project isn't using JSON functions, just the column/dtype, so we should be OK for now.